### PR TITLE
Add Add->Clamp->Div->Mul to HSwish fusion

### PR DESCRIPTION
**Pull request by mateusztabaka**

### CODE REVIEW

File: inference-engine/src/transformations/src/transformations/common_optimizations/hswish_fusion.cpp [hunk 1]
```diff
@@ -219,3 +219,48 @@ ngraph::pass::HSwishFusionWithClamp::HSwishFusionWithClamp() {
...
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::HSwishFusionWithClampDiv, "HSwishFusionWithClampDiv", 0);
+
+ngraph::pass::HSwishFusionWithClampDiv::HSwishFusionWithClampDiv() {
+    MATCHER_SCOPE(HSwishFusionWithClampDiv);
+    // Replaces a sub-graph x * (Clamp(x + 3, 0, 6) / 6) with a HSwish op.
+    auto input = ngraph::pattern::any_input();
+    auto add_constant = ngraph::pattern::wrap_type<ngraph::opset4::Constant>();
+    auto add = std::make_shared<ngraph::opset4::Add>(input, add_constant);
+    auto clamp = std::make_shared<ngraph::op::v0::Clamp>(add, 0.0f, 6.0f);
```
Comment (GlebKazantaev) [inline inference-engine/src/transformations/src/transformations/common_optimizations/hswish_fusion.cpp:231]:
```suggestion
    auto clamp = std::make_shared<ngraph::opыуе4::Clamp>(add, 0.0f, 6.0f);
```

File: inference-engine/src/transformations/src/transformations/common_optimizations/hswish_fusion.cpp [hunk 2]
```diff
@@ -219,3 +219,48 @@ ngraph::pass::HSwishFusionWithClamp::HSwishFusionWithClamp() {
...
+
+        hswish->set_friendly_name(m.get_match_root()->get_friendly_name());
+        ngraph::copy_runtime_info({ pattern_to_output.at(add_constant).get_node_shared_ptr(),
+                                    pattern_to_output.at(add).get_node_shared_ptr(),
+                                    pattern_to_output.at(clamp).get_node_shared_ptr(),
+                                    pattern_to_output.at(div_constant).get_node_shared_ptr(),
+                                    pattern_to_output.at(div).get_node_shared_ptr(),
+                                    pattern_to_output.at(mul).get_node_shared_ptr()
+                                  },
+                                  hswish);
```
Comment (GlebKazantaev) [inline inference-engine/src/transformations/src/transformations/common_optimizations/hswish_fusion.cpp:259]: You don't need to copy rt info for constants.
```suggestion
        ngraph::copy_runtime_info({pattern_to_output.at(add).get_node_shared_ptr(),
                                    pattern_to_output.at(clamp).get_node_shared_ptr(),
                                    pattern_to_output.at(div).get_node_shared_ptr(),
                                    pattern_to_output.at(mul).get_node_shared_ptr()
                                  },
                                  hswish);
```

File: inference-engine/tests/functional/inference_engine/transformations/hswish_fusion_test.cpp [hunk 3]
```diff
@@ -166,7 +166,38 @@ TEST(TransformationTests, HSwishFusionWithClamp) {
...
+    ASSERT_TRUE(res.first) << res.second;
+}
+
+TEST(TransformationTests, HSwishFusionWithClampDiv) {
+    std::shared_ptr<ngraph::Function> f(nullptr), f_ref(nullptr);
+    {
+        auto input = std::make_shared<ngraph::opset4::Parameter>(ngraph::element::f16, ngraph::PartialShape::dynamic(1));
+        auto add_constant = ngraph::opset4::Constant::create(ngraph::element::f16, ngraph::Shape{}, {3.0});
+        auto add = std::make_shared<ngraph::opset4::Add>(input, add_constant);
+        auto clamp = std::make_shared<ngraph::op::v0::Clamp>(add, 0.0f, 6.0f);
```
Comment (GlebKazantaev) [inline inference-engine/tests/functional/inference_engine/transformations/hswish_fusion_test.cpp:191]:
```suggestion
        auto clamp = std::make_shared<ngraph::opset4::Clamp>(add, 0.0f, 6.0f);
```

**Review by GlebKazantaev — CHANGES_REQUESTED** [review]

**Review by GlebKazantaev — APPROVED** [review]

**Review by mitruska — APPROVED** [review]

**Comment by ilyachur** [comment]
Please resolve merge conflicts

**Review by ilyachur — APPROVED** [review]

**Pull request merged**